Cargo.toml

```diff
@@ -4,12 +4,16 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-log = "0.4.17"
+bare-metal = "1.0.0"
+log        = "0.4.17"
+
+[dependencies.linked_list_allocator]
```

Comment:
Maybe we could enable the `spin` feature for multicore targets, like the esp32 & esp32s3?

Does something like this work?
```
[target.xtensa-esp32-none-elf.dependencies.linked_list_allocator]
features = ["spin"]
```

Otherwise people my inadvertently introduce race conditions without fully understanding their target.

Reply:
Ahh yeah good point, didn't think about this. I also didn't even consider the S2 or S3 :grin:

Unfortunately that form is unfortunately not supported as far as I know, but I am still able to enable these features for supported chips using somewhat more verbose syntax. Will update shortly with the fix.